flink-python/src/test/java/org/apache/flink/client/python/PythonEnvUtilsTest.java

```diff
@@ -76,10 +79,16 @@ public void testPreparePythonEnvironment() throws IOException {
         File relativeFile =
                 new File(tmpDirPath + File.separator + "subdir" + File.separator + "b.py");
         File schemeFile =
-                new File(tmpDirPath + File.separator + "subdir" + File.separator + "c.zip");
+                new File(tmpDirPath + File.separator + "subdir" + File.separator + "c.py");
```

Comment:
We can rename the "c.zip" to "c.egg" so that its expected python path would not be the same as "b.py".